### PR TITLE
1796 notes disaster food replacement, update to reflect new poli/temp guidance

### PR DESCRIPTION
**Pull request by MDR22**
Updated to ensure script captures all required details from 02/2024 POLI/TEMP for TE02.11.18 FOOD DESTROYED IN A MISFORTUNE OR DISASTER. Added TIKL functionality for DHS-1609. Updated dialog and validation. Added closing issue documentation and initial county and case number dialogs.

### CODE REVIEW

File: notes/disaster-food-replacement.vbs [hunk 1]
```diff
@@ -44,106 +44,200 @@ changelog = array()
...
+    IF replacement_status <> "Request Denied" and denial_reason <> "" THEN err_msg = err_msg & vbCr & "* The reason for the denial field should be blank if the request is not being denied."
+    IF replacement_status <> "Pending Verification(s)" and verif_needed <> "" THEN err_msg = err_msg & vbCr & "* The pending verifications field should be blank unless the 'Pending Verifications' decision on request option is selected."
+    IF rei_replacement = "Select One:" THEN err_msg = err_msg & vbCr & "* Please select if the replacement was REI, or select NA."
+    IF TSS_BENE_sent_checkbox = UNCHECKED and replacement_status = "Request Approved" THEN err_msg = err_msg & vbCr & "* Please check that the TSS BENE Webform has been completed."
+    IF TSS_BENE_sent_checkbox = CHECKED and replacement_status <> "Request Approved" THEN err_msg = err_msg & vbCr & "* Please only check the TSS BENE Webform checkbox if you are approving the request."
+    If ButtonPressed = TSS_BENE_webform_btn Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://owa.dhssir.cty.dhs.state.mn.us/csedforms/MMR/TSSBENE_BENE_request.asp"
```
Comment (MeganGeissler) [inline notes/disaster-food-replacement.vbs:161]: Error handling displays when this button is pressed. Not sure if this is intentional or not so thought I'd mention it.

File: notes/disaster-food-replacement.vbs [hunk 2]
```diff
@@ -44,106 +44,200 @@ changelog = array()
...
+  CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 Loop until are_we_passworded_out = false					'loops until user passwords back in
 
+'Create TIKL if selected
+If dhs_1609_tikl = CHECKED Then Call create_TIKL("DHS-1609 was sent 10 days ago and is now due back.", 10, date, False, TIKL_note_text)
 
+'Write CASE/NOTE with information
 start_a_blank_case_note
-'writes case note for Baby Born
 CALL write_variable_in_Case_Note("--Food Destroyed in a Disaster Reported - " & replacement_status & "--")
```
Comment (MeganGeissler) [inline notes/disaster-food-replacement.vbs:177]: Header of casenote could be shorter to show more of the replacement_status but okay as is.

**Review by MeganGeissler — APPROVED** [review]
Overall looks good- I like the format! Few minor suggestions.

File: notes/disaster-food-replacement.vbs (hunk 2 above)
Comment (MDR22) [reply notes/disaster-food-replacement.vbs]: Slightly shortened header so you can see more of the request status

**Review by MeganGeissler — APPROVED** [review]

**Review by IlseFerris — APPROVED** [review]
This looks great, I approve thee!

**Pull request merged**